packages/components/src/visually-hidden/component.js

```diff
@@ -1,8 +1,16 @@
 /**
  * Internal dependencies
  */
-import { createComponent } from '../ui/utils';
-import { useVisuallyHidden } from './hook';
+import { useContextSystem, contextConnect } from '../ui/context';
+import { VisuallyHidden as VisuallyHiddenWrapper } from './styles';
+
+/**
+ * @param {import('../ui/context').PolymorphicComponentProps<{ children: import('react').ReactNode }, 'div'>} props
+ */
+function VisuallyHidden( props ) {
+	const contextProps = useContextSystem( props, 'VisuallyHidden' );
+	return <VisuallyHiddenWrapper { ...contextProps } />;
```

Comment:
Maybe I'm just a bit confused, but — when the `as` prop is passed to `<VisuallyHidden />`, how is it used to actually render the component defined in the `as` prop? Previously, that was done via the `renderAsRenderProps` function

Reply:
Update: components created through the `styled` function accept the `as` prop.